app/views/admin/accounting/transactions/_show_modal_content.slim

```diff
@@ -0,0 +1,24 @@
+div.modal-header
```

Comment:
The modal content would use: `app/views/admin/accounting/transactions/_form.slim`. If you need help with this part, please let me know.

The best example I can see of a flow for this would be the project logs modal. The way the code is set up there would be what we would use similarly for this feature.

A show modal file would not be needed, if we can follow the same pattern.